src/components/Reown/SendTransactionModal.js

```diff
@@ -0,0 +1,439 @@
+
+const styles = StyleSheet.create({
+  body: {
+    paddingBottom: 24,
+    backgroundColor: COLORS.lowContrastDetail,
+  },
+  content: {
+    flex: 1,
+    rowGap: 24,
+    width: '100%',
+    paddingVertical: 16,
+    paddingHorizontal: 16,
+  },
+  sectionTitle: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginBottom: 16,
+  },
+  itemContainer: {
+    backgroundColor: COLORS.backgroundColor,
+    borderRadius: 12,
+    padding: 16,
+    marginBottom: 12,
+    shadowColor: COLORS.black,
+    shadowOffset: { width: 0, height: 2 },
+    shadowOpacity: 0.08,
+    shadowRadius: 4,
+    elevation: 2,
+  },
+  itemHeader: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    marginBottom: 12,
+    alignItems: 'center',
+  },
+  itemTitle: {
+    fontWeight: 'bold',
+    fontSize: 16,
+  },
+  valueText: {
+    fontSize: 16,
+    fontWeight: '600',
+  },
+  labelText: {
+    fontSize: 14,
+    color: 'hsla(0, 0%, 38%, 1)',
+    fontWeight: 'bold',
+    marginBottom: 4,
+  },
+  valueContainer: {
+    backgroundColor: COLORS.lowContrastDetail,
+    borderRadius: 8,
+    padding: 12,
+    marginBottom: 12,
+  },
+  monospace: {
+    fontFamily: 'Courier',
+    fontSize: 14,
+  },
+  copyButton: {
+    color: 'hsla(263, 100%, 64%, 1)',
+    marginTop: 4,
+  },
+  addressContainer: {
+    backgroundColor: COLORS.lowContrastDetail,
+    borderRadius: 8,
+    padding: 12,
+    marginBottom: 12,
+  },
+  actionContainer: {
+    flexDirection: 'column',
+    gap: 8,
+    paddingBottom: 24,
+  },
+  modalTitle: {
+    fontSize: 20,
+    fontWeight: 'bold',
+    textAlign: 'center',
+  },
+  loadingContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 24,
+  },
+  loadingText: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginTop: 16,
+    textAlign: 'center',
+  },
+  statusContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 24,
+  },
+  statusText: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginTop: 16,
+    textAlign: 'center',
+  },
+  statusMessage: {
+    fontSize: 14,
+    marginTop: 8,
+    textAlign: 'center',
+    color: 'hsla(0, 0%, 38%, 1)',
+  },
+  successColor: {
+    color: 'green',
+  },
+  errorColor: {
+    color: 'red',
+  },
+});
+
+export default ({
+  onDismiss,
+  data,
+  onAcceptAction,
+  onRejectAction,
+}) => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const { tokens: registeredTokens } = useSelector((state) => ({
+    tokens: state.tokens,
+  }));
+
+  const getTokenSymbol = (tokenId) => {
+    if (!tokenId) {
+      return constants.DEFAULT_NATIVE_TOKEN_CONFIG.symbol;
+    }
+
+    const token = registeredTokens[tokenId];
+    if (token) {
+      return token.symbol;
+    }
+
+    return '?';
+  };
+
+  const formatValue = (value) => {
+    if (value == null) {
+      return '-';
+    }
+
+    return numberUtils.prettyValue(value);
```

Comment:
We should format considering NFT as well.

Reply:
Done, thanks